src_py/apiServer/apiServer.py

```diff
@@ -41,19 +31,47 @@ def getQueueData(self):
         
         while not received:
             if not multiProcQueue.empty():
-                print("Loss map created!")
-                lossMap = multiProcQueue.get()
-                print(lossMap)
-                return lossMap      
+                print("New loss map created!")
+                multiProcQueue.get()
+                received = True
```

Comment:
Do you always receive just a single element from multiProcQueue? 
The received flag limits you to a single elements.
@dordor7